README.md

```diff
@@ -127,6 +127,64 @@ client = Redis.new(semian: {
 })
 ```
 
+#### Warning: Thread unsafe
+
+Semian is currently known to be thread-unsafe, and is intended to be used by *separate processes*.
+
+Internally, semian uses ```SEM_UNDO``` for several sysv semaphore operations:
```

Comment:
Single backticks are adequate here, could you use single backticks around other mentions of `SEM_UNDO` for completeness?